gotocompany/compass/v1beta1/service.proto

```diff
@@ -816,6 +832,52 @@ message SearchAssetsRequest {
   }];
 }
 
+
+message GroupAssetsRequest {
+
+  string groupby = 1 [
+    (validate.rules).string = {
+      ignore_empty: false,
+    },
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "fields used to group assets. multiple fields can also be provided here"
+    }
+  ];
+
+  map<string, string> filter = 2 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
```

Comment:
> For instance, to specify two landscape 'vn' and 'th', the query could be `/groupassets/?groupby=<groupby>&filter[environment]=integration&filter[landscape]=vn&filter[landscape]=th`.

Will this really work? Wouldn't we need to specify a CSV for it to be compatible with a single string field?

Reply:
Could you elaborate more on this, I could not understand the concern here.

Reply:
Here, if you specify `&filter[landscape]=vn&filter[landscape]=th` in the query string, it translates to `filter[environment]=[vn, th]`. So am not sure how this gets translated into a `string` field inside the map.

Reply:
The user is expected to give the values as filter[environment]=vn,th in such a case. This is how we are doing in case of SearchAssetsRequest as well.

Reply:
The statement I quoted is from the OpenAPI description for the rpc. Could you amend the same?